modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/SCIM2RestClient.java

```diff
@@ -156,7 +161,27 @@ public JSONObject getUser(String userId, String attribute) throws Exception {
      */
     public JSONObject getSubOrgUser(String userId, String switchedM2MToken) throws Exception {
 
-        String endPointUrl = getSubOrgUsersPath() + PATH_SEPARATOR + userId;
+        return getSubOrgUser(userId, null, switchedM2MToken);
+    }
+
+    /**
+     * Get the details of a user of a sub organization.
+     *
+     * @param userId           ID of the user.
+     * @param attribute Requested user attributes.
+     * @param switchedM2MToken Switched M2M token for the given organization.
+     * @return JSONObject of the HTTP response.
+     * @throws Exception If an error occurred while getting a user.
+     */
+    public JSONObject getSubOrgUser(String userId, String attribute, String switchedM2MToken) throws Exception {
```

Comment:
```suggestion
    public JSONObject getSubOrgUser(String userId, String attributes, String switchedM2MToken) throws Exception {
```

Reply:
fixed in [3b01ec0](https://github.com/wso2/product-is/pull/23064/commits/3b01ec03b91f4889e6dce7f7147274ce5ca3da6c)